utils/dataloaders.py

```diff
@@ -65,29 +73,40 @@ def get_fashion_mnist_dataloaders(batch_size=128, shuffle=True,
     return train_loader, test_loader
 
 
-def get_dsprites_dataloader(batch_size=128, shuffle=True,
-                            path_to_data=os.path.join(DIR, '../data/datadsprites/dsprites_data.npz')):
+def get_dsprites_dataloader(batch_size=128,path_to_data=os.path.join(DIR, '../data/dsprites-dataset'), shuffle=True):
     """DSprites dataloader."""
-    dsprites_data = DSpritesDataset(path_to_data,
+    print('in loop  ')
```

Comment:
It takes a long time to download. 3DChairs is the uncropped data and 3DChairs_64 is the cropped data that we use. I would like to save both folders in case we want to investigate the original pictures, etc.

Reply:
OK I agree that it is good to keep both datasets in that case. Removing the print or using a more intuitive message would still be nice.

Reply:
@BartMelman never use print statements, use `logging`. Check out how it's done in `training.py`. Logging is better because you can deactivate all messages at once if you don't want them. Please be more precise than "in loop" though. E.g. "Loading dsprites"